src/tactic/lint.lean

```diff
@@ -744,14 +744,15 @@ meta def lint_aux (l : list declaration) (group_by_filename : option nat)
   tactic (name_set × format) := do
 results ← lint_core l checks,
 formatted_results ← results.mmap $ λ ⟨linter_name, linter, results⟩,
+  let report_str : format := to_fmt "/- The `" ++ to_fmt linter_name ++ "` linter reports: -/\n" in
   if ¬ results.empty then do
     warnings ← match group_by_filename with
       | none := print_warnings results
       | some dropped := grouped_by_filename results dropped print_warnings
       end,
-    pure $ to_fmt "/- " ++ linter.errors_found ++ ": -/\n" ++ warnings
+    pure $ report_str ++ "/- " ++ linter.errors_found ++ ": -/\n" ++ warnings
   else
-    pure $ if verbose then "/- OK: " ++ linter.no_errors_found ++ ". -/" else format.nil,
+    pure $ if verbose then report_str ++ "/- OK: " ++ linter.no_errors_found ++ ". -/" else format.nil,
```

Comment:
```suggestion
    pure $ if verbose then "/- OK: " ++ linter.no_errors_found ++ ". -/" else format.nil,
```
How about we only print the linter name if it finds an error? When the linter is happy we won't have any reason to care what it is called.

Reply:
I figured the consistency was worth a few extra lines in the linter output, but I don't particularly care. Anyone else have an opinion? If not I'll change it.

Reply:
I slightly prefer the shorter version (i.e., Scott's suggestion).